stdlib/public/runtime/Errors.cpp

```diff
@@ -170,7 +170,8 @@ void swift::dumpStackTraceEntry(unsigned index, void *framePC,
   if (foundSymbol) {
     offset = ptrdiff_t(uintptr_t(framePC) - symbolAddr);
   } else {
-    offset = ptrdiff_t(uintptr_t(framePC) - uintptr_t(syminfo.baseAddress));
+    auto baseAddress = syminfo.getBaseAddress();
+    offset = ptrdiff_t(uintptr_t(framePC) - uintptr_t(baseAddress));
```

Comment:
Not your doing, but while here, makes sense to switch to C++ style ...

```suggestion
    offset = static_cast<ptrdiff_t>(static_cast<uintptr_t>(framePC) - static_cast<uintptr_t>(baseAddress));
```

Reply:
I think I'm going to leave this alone for now—this whole function could be rewritten to be half as many lines anyway.

I was thinking of adding a `getDisplacement()` function to `SymbolInfo` anyway. If I add that, then this `offset` field is just that value and doesn't need any additional magic.